src/Init/Data/Int/DivMod/Lemmas.lean

```diff
@@ -187,6 +187,9 @@ theorem tdiv_eq_ediv_of_nonneg : ∀ {a b : Int}, 0 ≤ a → a.tdiv b = a / b
   | succ _, succ _, _ => rfl
   | succ _, -[_+1], _ => rfl
 
+@[simp] theorem ofNat_tdiv_eq_ediv {a : Nat} {b : Int} : (a : Int).tdiv b = a / b :=
```

Comment:
This lemma should be called `natCast_tdiv_eq_ediv` (Note: there are a _lot_ of incorrect uses of the naming component `ofNat` in core. We'll get there.).

Reply:
Fixed. Thank you for the timely review.